### ISSUE TITLE
Switch to pytest.

### ISSUE THREAD

**Issue by airelil** [pywinauto/pywinauto#306]
Should we switch to [pytest](http://docs.pytest.org/en/latest/) from unittest? We mentioned it once, in #138. The pros I see: the immediate fallback in pdb on a test failure, parametirezed tests. Possibly  [fixtures](http://docs.pytest.org/en/latest/fixture.html#fixture) too.

**Comment by vasily-v-ryabov** [comment]
Generally I don't mind, but it's not a high priority for me. Would be nice to have some tests ported to pytest to see how it makes difference.

**Comment by airelil** [comment]
I'll setup a separate branch in my repo to play with it....

**Comment by airelil** [comment]
I switched Appveyor to pytest in my '[pytest](https://github.com/airelil/pywinauto/tree/pytest)' branch. The output example can be found at: https://ci.appveyor.com/project/airelil/pywinauto/build/1.0.63

**Comment by vasily-v-ryabov** [comment]
I see you just switched to pytest as a runner. It looks good to me.
I thought you had plans to re-write the code from unittest to pytest as well. Probably it's a next step. Am I right?

**Comment by airelil** [comment]
Yes, it was a first step. The next one will be try to switch several tests and see if there is any added value of it. It would be nice to see if these tests still run smoothly in your favourite Python IDE
